ui/src/attestation/contract.ts

```diff
@@ -53,24 +53,45 @@ export class ClaimsContract {
     transaction.add(transaction.claimRadicleIdentity(tx, urn));
   }
 
+  // Fetches the identity claimed by the given Ethereum address.
+  // Returns `undefined` if currently there's no valid claim.
+  async getClaimed(address: string): Promise<Uint8Array | undefined> {
+    const filter = this.contract.filters.Claimed(address);
+    const lastEvent = (await this.contract.queryFilter(filter)).pop();
+    if (!lastEvent) {
+      return undefined;
+    }
+    let claimed;
+    try {
+      claimed = await this.getClaimedByTx(lastEvent.transactionHash, address);
+    } catch {
+      return undefined;
```

Comment:
Is there a reason we swallow errors?

Reply:
We don't, we return `undefined` to signal that there's no valid claim on the Ethereum side. Added a comment.